diff --git a/node_modules/expo-eas-client/ios/EASClient/EASClientModule.swift b/node_modules/expo-eas-client/ios/EASClient/EASClientModule.swift
index 9f80ef8..3a94729 100644
--- a/node_modules/expo-eas-client/ios/EASClient/EASClientModule.swift
+++ b/node_modules/expo-eas-client/ios/EASClient/EASClientModule.swift
@@ -6,6 +6,7 @@ public class EASClientModule: Module {
   public func definition() -> ModuleDefinition {
     Name("EASClient")
 
-    Constant("clientID") { EASClientID.uuid().uuidString }
+    // Constant API Expo SDK 52'de değişti, geçici olarak devre dışı
+    // Constant("clientID") { EASClientID.uuid().uuidString }
   }
 }
